app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/AcceptOfferDialog.scala

```diff
@@ -242,8 +243,8 @@ class AcceptOfferDialog extends CliCommandProducer[AcceptDLCOffer] {
             dlcDetailsShown = true
             showOfferTerms(lnMessage.tlv)
             offerTLVTF.editable = false
-            if (dialog != null)
-              dialog.dialogPane().getScene.getWindow.sizeToScene()
+            if (dialogOpt.isDefined)
+              dialogOpt.get.dialogPane().getScene.getWindow.sizeToScene()
```

Comment:
rather than doing this it is better to do a `match` on `dialogOpt` and handle both cases

Reply:
I left all the dialogOpt.isDefineds in place. These only apply to the case where a dialog is getting used and there is no extra value in the 'else' case which automatically sizes